tests/server_test.go

```diff
@@ -9857,6 +9858,111 @@ func TestServer_Prometheus_Write(t *testing.T) {
+----------------------  ----------------------  ----------------------  ------------------------------  --------------------------  
+                     f                      m0                      k1  1970-01-01T00:00:00.000000005Z                           5  
+                     f                      m0                      k1  1970-01-01T00:00:00.000000006Z                           6  
+Table: keys: [_field, _measurement, k]
+         _field:string     _measurement:string                k:string                      _time:time                  _value:int  
+----------------------  ----------------------  ----------------------  ------------------------------  --------------------------  
+                     f                      m1                      k0  1970-01-01T00:00:00.000000007Z                           5  
+                     f                      m1                      k0  1970-01-01T00:00:00.000000009Z                           6  
+                     f                      m1                      k0  1970-01-01T00:00:00.000000011Z                           7  
+                     f                      m1                      k0  1970-01-01T00:00:00.000000012Z                           5  
+Table: keys: [_field, _measurement, k]
+         _field:string     _measurement:string                k:string                      _time:time                  _value:int  
+----------------------  ----------------------  ----------------------  ------------------------------  --------------------------  
+                     f                      m1                      k1  1970-01-01T00:00:00.000000010Z                           6  
+                     f                      m1                      k1  1970-01-01T00:00:00.000000013Z                           8  
+Table: keys: [_field, _measurement, k]
+         _field:string     _measurement:string                k:string                      _time:time                  _value:int  
+----------------------  ----------------------  ----------------------  ------------------------------  --------------------------  
+                     f                      m1                      k2  1970-01-01T00:00:00.000000008Z                           0  
+                     f                      m1                      k2  1970-01-01T00:00:00.000000014Z                           9  
+Table: keys: [_field, _measurement, k]
+         _field:string     _measurement:string                k:string                      _time:time                  _value:int  
+----------------------  ----------------------  ----------------------  ------------------------------  --------------------------  
+                     f                      m1                      k3  1970-01-01T00:00:00.000000015Z                           5  
+`
+	// No guaranteed order for results to come back in
```

Comment:
Is the ordering guaranteed within each `yield`?

Reply:
That's - actually a good question. I've run it quite a few times and not seen any failures, and they do seem to be coming back in series key order. I think it's guaranteed by the current TSM implementation. If that changes or we see failures I can update the test.